python/pyspark/sql/connect/functions.py

```diff
@@ -89,6 +93,505 @@ def lit(col: Any) -> Column:
     return Column(LiteralExpression(col))
 
 
+# def bitwiseNOT(col: "ColumnOrName") -> Column:
+#     """
+#     Computes bitwise not.
+#
+#     .. versionadded:: 1.4.0
+#
+#     .. deprecated:: 3.2.0
+#         Use :func:`bitwise_not` instead.
+#     """
+#     warnings.warn("Deprecated in 3.2, use bitwise_not instead.", FutureWarning)
+#     return bitwise_not(col)
```

Comment:
Need to remove this?

Reply:
I prefer just letting it alone for now, to make sure nothing is missing for now.